classes/dpkg-bazel.bbclass

```diff
@@ -13,7 +14,7 @@ inherit dpkg
 DEPENDS += "bazel-bootstrap"
 
 # fetch all bazel deps and add to package sources
-do_fetch_bazel_deps() {
+bazel_fetch() {
     dpkg_do_mounts
     E="${@ isar_export_proxies(d)}"
     sudo chroot --userspec=$( id -u ):$( id -g ) ${BUILDCHROOT_DIR} /bin/bash -c 'cd ${PP}/${PPS} && rm -rf local && ./debian/rules local'
```

Comment:
We should also consider renaming the `local` target to `get-orig-source`. While this is deprecated for use-cases that can be handled using uscan, we still can use it here as the situation is more complex. [1]

- [1]: https://wiki.debian.org/onlyjob/get-orig-source

Reply:
Yes, but that renaming can happen on top.